content/fr/articles/css/stop-css/index.md

```diff
@@ -0,0 +1,233 @@
+---
+date: "2016-06-16"
+title: Pourquoi j'ai arrêté d'utiliser CSS
+tags:
+  - css
+  - style-inline
+  - javascript
+authors:
+  - bloodyowl
+---
+
+On va commencer en douceur : CSS, c'est de la grosse merde.
```

Comment:
Maybe you can found something more subtle:
- CSS est un langage du passé, qui va finir par pourrir dans une placard, par là
- CSS c'est tout l'inverse des SS,  pas du tout stricte et beaucoup trop flexible
- CSS illustre parfaitement la fameuse phrase «Un grand pouvoir implique de grandes responsabilités» , prononcée par l'oncle de Peter Parker. Phrase qui peut évoquer Socrate et d'autres philosophes traitant le difficile problème du pouvoir. Du coup faut bien avour, on peut rapidement faire de la grosse merde avec CSS et c'est bien trop souvent ce qui se passe.
- utiliser CSS c'est un peu comme essayer de faire du composte de la merde de chien, ça va pas marcher.

Well it's just some idea, I am sure you can find something better ;)
Our readers deserve it.